src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -27,6 +27,17 @@
 #include "src/core/lib/gprpp/ref_counted.h"
 #include "src/core/lib/security/security_connector/ssl_utils.h"
 
+struct grpc_tls_err_details
+    : public grpc_core::RefCounted<grpc_tls_err_details> {
+ public:
+  void set_err_details(const char* err_details) { error_msg_ = err_details; }
+  std::string err_details() { return error_msg_; }
+  bool has_error() { return error_msg_.empty(); }
```

Comment:
`has_error` should be true when `error_msg` is not empty, right? This is exact opposite.

Reply:
fixed in the new PR.